vrx_gazebo/src/vrx_gazebo/compliance.py

```diff
@@ -183,7 +183,7 @@ def find_boxes(box_yaml):
 
```

Comment:
In lines 157 and 172-177 above, are we intending to use integer division? It might not matter, but I ust want to make sure...